src/widget/contentlayout.cpp

```diff
@@ -71,15 +71,11 @@ void ContentLayout::clear()
     QLayoutItem* item;
     while ((item = mainHead->layout()->takeAt(0)) != 0)
     {
-        item->widget()->hide();
-        item->widget()->setParent(nullptr);
```

Comment:
Guess you're talking about that "overlayed" layout. Yes, that's fine. The commit doesn't fix everything, but  represents an interims state (which obviously is even compilable, yay :)). It doesn't make sense to fix a bug, which actually does not exist. Effort would simply be huge to compile and check every time…